src/util/AppContextUtil/Shogun2AppContextUtil.tsx

```diff
@@ -514,6 +514,7 @@ class Shogun2AppContextUtil extends BaseAppContextUtil implements AppContextUtil
             iconName="fas fa-expand"
             tooltip={t('ZoomToExtent.tooltip')}
             tooltipPlacement={'right'}
+            constrainViewResolution={true}
```

Comment:
should this be added in the other context parsers as well?

Reply:
Right, added to `ShogunBootApplicationContextUtil`